com.github.Alcaro.Flips.json

```diff
@@ -0,0 +1,43 @@
+{
+    "app-id" : "com.github.Alcaro.Flips",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "3.34",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "flips",
+    "finish-args" : [
+        /* X11 + XShm access */
+        "--share=ipc",
+        "--socket=x11",
+        /* Wayland access */
+        "--socket=wayland",
+        /* OpenGL + DRI access */
+        "--device=dri",
+        /* Needed to find the ROMs */
+        "--filesystem=host:ro"
+    ],
+    "build-options" : {
+        "cflags" : "-O2 -g",
+        "cxxflags" : "-O2 -g",
+        "env" : {
+        }
+    },
```

Comment:
Already set in the runtime.

Reply:
Do you know if it's also set on master? So I can remove it whenever I see it elsewhere. :)

Reply:
They are set on fdo, so in GNOME master too and everything that derives from it.

Reply:
Perfect, thanks! \o/